src/index.js

```diff
@@ -21,6 +21,7 @@ export { default as Legend } from './Legend'
 export { default as LinearProgress } from './LinearProgress'
 export { default as Pagination } from './Pagination'
 export { default as RadioGroup } from './RadioGroup'
+export { default as SecondLinearProgress } from './LinearProgress/SecondLinearProgress'
```

Comment:
Mudaria o nome do componente pra manter o padrão do que foi decidido na outra thread :slightly_smiling_face: 

```suggestion
export { default as SecondaryLinearProgress } from './LinearProgress/SecondaryLinearProgress'
```

Reply:
poutz, nem percebi que tinha colocado `Second` ao inves de `Secondary`, vou alterar aqui :heavy_check_mark:

Reply:
Fixed :heavy_check_mark:. Valeu @vagnervst!